src/test/fuzz/package_eval.cpp

```diff
@@ -301,5 +302,21 @@ FUZZ_TARGET(tx_package_eval, .init = initialize_tx_pool)
     UnregisterSharedValidationInterface(outpoints_updater);
 
     WITH_LOCK(::cs_main, tx_pool.check(chainstate.CoinsTip(), chainstate.m_chain.Height() + 1));
+    LOCK(tx_pool.cs);
+    for (const auto& tx_info : tx_pool.infoAll()) {
+        const auto& entry = *Assert(tx_pool.GetEntry(tx_info.tx->GetHash()));
+        if (tx_info.tx->nVersion == 3) {
+            // Check that special v3 ancestor/descendant limits and rules are always respected
+            Assert(entry.GetCountWithDescendants() <= V3_DESCENDANT_LIMIT);
+            Assert(entry.GetCountWithAncestors() <= V3_ANCESTOR_LIMIT);
+            if (entry.GetCountWithAncestors() > 1) {
+                Assert(entry.GetTxWeight() <= V3_CHILD_MAX_WEIGHT);
+            }
+        }
+        // Transactions with fees of 0 or lower should be proactively trimmed.
+        if (tx_pool.m_min_relay_feerate.GetFeePerK() > 0) {
+            Assert(entry.GetModFeesWithDescendants() > 0);
```

Comment:
If `bypass_limits` is set for a single submission, this can fail since no trimming is done after that.

Reply:
Ah true. I forgot that we do an atmp here too. I don't think we should do `bypass_limits` though - wouldn't that cause the result to deviate from the package eval result?

Reply:
yeah I don't think we need to ever

Reply:
Made bypass_limits always false